lib/ansible/modules/network/aci/aci_switch_leaf_policy_profile.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_switch_leaf_policy_profile
+short_description: Create switch policy leaf profiles on Cisco ACI fabrics (infra:NodeP)
+description:
+- Create switch policy leaf profiles on Cisco ACI fabrics.
+- More information from the internal APIC class
+  I(infra:NodeP) at U(https://developer.cisco.com/site/aci/docs/apis/apic-mim-ref/).
+author:
+- Bruno Calogero (@brunocalogero)
+version_added: '2.5'
+options:
+  switch_profile:
+    description:
+    - The name of the Switch Profile.
+    required: yes
+    aliases: [ switch_profile_name ]
```

Comment:
This should also include the alias `name` as all of the ACI modules that create an object also has the alias  `name` defined.

Reply:
ook